### PR TITLE
Link2defaults

### PR DESCRIPTION
**Pull request by ryestew**
fixes https://github.com/ethereum/remix-project/issues/843

### CODE REVIEW

**Comment by ryestew** [comment]
The PR updates the images of the new file explorer and updates instructions for installing remixd.

File: docs/file_explorer.md [hunk 1]
```diff
@@ -5,11 +5,14 @@ To get to the File Explorers module - click the file explorers icon.
 
 ![](images/a-file-explorer1.png)
 
-The basic files explorer lists all the files stored in your browser's **browser storage**.
-You can see them in the **browser** folder. 
+By default Remix IDE **ONLY** stores files in your browser's local storage.  It is possible to go beyond this default behavior - see the **Important Note** below.
+
+The files explorer's **browser folder** has a sample project in it. The files stored in your browser's **browser storage**.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:10]: The files _are_ stored in...

File: docs/file_explorer.md [hunk 2]
```diff
@@ -5,11 +5,14 @@ To get to the File Explorers module - click the file explorers icon.
...
+
+If, when you open Remix IDE, you do not see the sample project (like in the image above), they will appear when all the files in the File Explorer are erased or when the cache is cleared.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:12]: Choose one from `If, when`

File: docs/file_explorer.md [hunk 3]
```diff
@@ -25,35 +28,54 @@ The icons to the right of the **browser** file explorer in the image above only
...
 The functionality of the context menu also works with RemixD (which gives you have access to a folder on your hard drive).  
 
 **Note:** When working with RemixD, you need to open and close the **localhost** folder to refresh the view.
+
+Right Click on a Script
+------------------------
+
+![](images/a-file-ex-rt-click-script.png)
+
+Right click on any file with a .js extension will gives the **run** option in the context menu.  The **run** shortcut is equivalent to getting the script file to be the active tab in the editor and then running the command `remix.exeCurrent()` in the console.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:81]: will give

**Review by Aniket-Engg — CHANGES_REQUESTED** [review]

**Review by LianaHus — APPROVED** [review]

**Pull request merged**